### ISSUE TITLE
TLS Error: TLS key negotiation failed to occur within 60 seconds (check your network connectivity)

### ISSUE THREAD

**Issue by infosecuz** [Angristan/OpenVPN-install#1272]
![image](https://github.com/user-attachments/assets/12c17d9d-b174-4d12-a229-445cbb9018d9)
![image](https://github.com/user-attachments/assets/5382f672-1257-414a-b010-3b99c572f999)

My linux information : Linux vmi2096352.contaboserver.net 5.15.0-128-generic #138-Ubuntu SMP Sat Nov 30 22:28:23 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

HOW TO SOLVE THIS ERROR ?

**Comment by RoderikTheRed** [comment]
I am facing the same issue, have you found a solution?

**Comment by Petrprogs** [comment]
Same issue, but only on android (Xiaomi 9c) . Network manager on Manjaro works like a charm

**Comment by infosecuz** [comment]
> I am facing the same issue, have you found a solution?

yes i've just deleted all files and this vpn, instead of this i'm using access server of OpenVPN, this is great !!

**Comment by Petrprogs** [comment]
> > I am facing the same issue, have you found a solution?
> 
> yes i've just deleted all files and this vpn, instead of this i'm using access server of OpenVPN, this is great !!

So I found solution too. I'm just revoked all the clients and created them again

**Comment by farrooh** [comment]
I can connect from my laptop but am getting this exact error (on server) and Connection timeout error (on mobile) when I try to connect from my Android (v14) phone. Any ideas why this could be happening?

Edit:
My phone sometimes can connect, but there is no internet connection when connected.

**Comment by Petrprogs** [comment]
> I can connect from my laptop but am getting this exact error (on server) and Connection timeout error (on mobile) when I try to connect from my Android (v14) phone. Any ideas why this could be happening?
> 
> Edit:
> My phone sometimes can connect, but there is no internet connection when connected.

Stupid question, but anyway. Do you using separate configs for your phone and pc?

**Comment by farrooh** [comment]
> > I can connect from my laptop but am getting this exact error (on server) and Connection timeout error (on mobile) when I try to connect from my Android (v14) phone. Any ideas why this could be happening?
> > Edit:
> > My phone sometimes can connect, but there is no internet connection when connected.
> 
> Stupid question, but anyway. Do you using separate configs for your phone and pc?

Yes, there are 2 ovpn files, and I am not using the PC config file on my phone.

**Comment by rzayoruk** [comment]
anybody?
